include/ensmallen_bits/de/de_impl.hpp

```diff
@@ -110,6 +114,15 @@ inline double DE::Optimize(DecomposableFunctionType& function,
       population.slice(member) = iterate;
     }
 
+    // Check for termination criteria.
+    if (lastBestFitness - fitnessValues.min() < objectiveChange)
```

Comment:
I think we can just use ``threshold`` here, as we did for the SGD optimizer:

https://github.com/mlpack/ensmallen/blob/964b6801ac945b60907c9f3b27b9adac3275f68e/include/ensmallen_bits/sgd/sgd_impl.hpp#L93-L98

Reply:
Seeing as I implemented the termination criteria the same way as in CNE, should i change that as well?

Reply:
Yes, thanks.